Movie/Movie/Core/CoreData/Models/CoreDataMovie+CoreDataProperties.swift

```diff
@@ -0,0 +1,23 @@
+// CoreDataMovie+CoreDataProperties.swift
+// Copyright © RoadMap. All rights reserved.
+
+import CoreData
+import Foundation
+
+/// CoreDataMovie
```

Comment:
Описание не пишется camelCase
```suggestion
/// Core Data Movie
```